shared/constants/search.js

```diff
@@ -7,10 +7,10 @@ export type SearchPlatforms = 'Keybase' | 'Twitter' | 'Github' | 'Reddit' | 'Coi
 
 export type ExtraInfo = {
   service: 'external',
-  icon: IconType,
+  icon: ?IconType,
   serviceUsername: string, // i.e. with twitter it would be malgorithms
-  serviceAvatar: ?string, // i.e. with twitter it would be their twitter avatar url
-  fullNameOnService: ?string // Say with twitter we know malgorithms is "Chris Coyne"
+  serviceAvatar: string, // i.e. with twitter it would be their twitter avatar url
```

Comment:
`''` is a string, but semantically it means we don't have that data. I'm just leaning to making it explicit that we could not have the data versus implying we won't have that data as an empty string